### PR TITLE
chore: 404 page tweaks

### PR DESCRIPTION
**Pull request by chalin**
Staged at https://angulardart-org-dev.firebaseapp.com/404

### CODE REVIEW

File: src/404.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: 404 Error
+description: webdev.dartlang's 404 page.
+sitemap: false
+---
+
+The page you requested doesn't exist.
+Things to try:
+
+- Check the URL's spelling.
+- Use the Search field to look for the page.
```
Comment (kwalrath) [inline src/404.md]: Search -> **Search**

**Comment by chalin** [comment]
Oh, did the policy change? I seem to recall that the recommendation (at least for button names) was to use italics rather than bold.

Anyhow, shall I make "Back" in "Click your browser’s Back button" in bold too?

**Comment by chalin** [comment]
> Anyhow, shall I make "Back" in "Click your browser’s Back button" in bold too?

I'll assume so.

**Comment by kwalrath** [comment]
> > Anyhow, shall I make "Back" in "Click your browser’s Back button" in bold too?
> 
> I'll assume so.

Yes? I wasn't sure. :)

**Pull request merged**